bump_version_doc/action.yml

```diff
@@ -0,0 +1,55 @@
+name: Bump version for doc
+description: Bump version for doc
+inputs:
+  new_version:
+    description: New version
+    required: true
+  version_file:
+    description: Version file
+    default: exchange.json
+  committer:
+    description: Committer for the pull request
+    required: true
+  assignees:
+    description: Assignees for the pull request
+    required: true
+  github_token:
+    description: GitHub token to create the pull request
+    required: true
```

Comment:
should we move this into the `secret` instead of `input` @hoangmirs

Reply:
The composite action does not have the `secrets` attribute, all values will be passed as inputs 🙏

Reply:
I see @hoangmirs